framework/core/js/src/forum/components/HeaderPrimary.tsx

```diff
@@ -16,7 +16,8 @@ export default class HeaderPrimary extends Component {
    *
    * @return {ItemList<import('mithril').Children>}
    */
-  items() {
-    return new ItemList();
+  
+  items(): ItemList<import('mithril').Children> {
+    return new ItemList<import('mithril').Children>();
```

Comment:
let's iimport Mithril at the top like we do elsewhere, then use that

```ts
import type Mithril from 'mithril";
```

then use `Mithril.Children`

Reply:
Hi @SychO9 changes are committed please have a look😊

Thanks
Vijayan K S